packages/menu/src/Menu.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useCallback, EventHandler } from 'react';
+import React, { useState, useRef, SetStateAction } from 'react';
```

Comment:
We have precedent to use React types like `React.SetStateAction`, rather than using import destructuring.

Reply:
👍